pkg/apiserver/storage/ram/watch.go

```diff
@@ -25,6 +25,18 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/apiserver/storage"
 )
 
+type bookMarkEvent struct {
+	ResourceVersion uint64
+}
+
+func (b *bookMarkEvent) ToWatchEvent(selectors *storage.Selectors, isInitEvent bool) *watch.Event {
+	return &watch.Event{Type: watch.Bookmark}
```

Comment:
is it unusual for the `Object` field to be nil here?
according to the [doc](https://godoc.org/k8s.io/apimachinery/pkg/watch#Event):
> If Type is Bookmark: the object (instance of a type being watched) where only ResourceVersion field is set. On successful restart of watch from a bookmark resourceVersion, client is guaranteed to not get repeat event nor miss any events.

It seems a bit strange to use a k8s `Bookmark` event without including the resource version, no? even if it's not leveraged by the current client code.

Reply:
I'm open to set it, but wonder whether we should set it for all events, also discussed here:  https://github.com/vmware-tanzu/antrea/pull/704#issuecomment-627308720